src/Microsoft.DotNet.Watcher.Tools/CommandLineOptions.cs

```diff
@@ -14,10 +14,7 @@ internal class CommandLineOptions
         public IList<string> RemainingArguments { get; private set; }
         public static CommandLineOptions Parse(string[] args, TextWriter consoleOutput)
         {
-            if (args == null)
-            {
-                throw new ArgumentNullException(nameof(args));
-            }
+            Ensure.NotNull(args, nameof(args));
```

Comment:
[EF does this.](https://github.com/aspnet/EntityFramework/blob/dev/src/Shared/Check.cs). I like the terseness, and I'mnNot too worried about the stack trace issue as this is "product code" in the sense that dotnet-watch isn't a class library people will use. But if you'd still rather we don't use this pattern here, I can revert.